lms/static/sass/elements/_controls.scss

```diff
@@ -323,3 +323,118 @@
       margin-bottom: none;
     }
   }
+
+
+// ===================
+// Dropdown menus
+// ===================
+.wrapper-more-actions {
+    @include float(right);
+    @include margin-right(($baseline/2));
+    position: relative;
+    display: inline-block;
+
+    .button-more.has-dropdown {
+        @extend %ui-depth2;
+        @include transition(all, $tmg-f2);
+        position: relative;
+        padding: ($baseline/2);
+        border: 1px solid transparent;
+        background: $white;
+        box-shadow: none;
+        font-size: $body-font-size;
+        text-shadow: none; // Clears text shadows on all buttons
+        line-height: ($baseline/2);
+
+        .icon {
+            @include transition(all, $tmg-f2);
+            @include transform-origin(45% center); // odd first value to keep icons centered
```

Comment:
Hmm, unfortunately that didn't work either. The issue is with the size of the `:before` (the actual icon). See this screenshot:
![screen shot 2015-05-15 at 12 48 37 pm](https://cloud.githubusercontent.com/assets/2112024/7657547/c05bc9ce-fb00-11e4-8de2-66a9e65434c6.png)

You can see that the icon itself is 8x14 pixels, which makes it rotate funny. Any other ideas?